lib/active_model/serializer/reflection.rb

```diff
@@ -14,6 +14,19 @@ class Serializer
     #     end
     #     has_many :secret_meta_data, if: :is_admin?
     #
+    #     has_one :blog do |serializer|
+    #       meta count: object.roles.count
+    #       serializer.cached_blog
+    #     end
+    #
+    #     private
+    #
+    #     def cached_blog
+    #       cache_store.fetch("cached_blog:#{object.updated_at}") do
+    #         Blog.find(object.blog_id)
+    #       end
+    #     end
```

Comment:
I added a bunch of code comments here and elsewhere in the file re the actually interface since I'm always finding them a little hard to track down.. not sure if they're all good example or even belong here